homeassistant/components/actiontec/device_tracker.py

```diff
@@ -12,17 +12,13 @@
     DeviceScanner,
 )
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
 import homeassistant.helpers.config_validation as cv
-import homeassistant.util.dt as dt_util
+from homeassistant.helpers.typing import ConfigType
 
-_LOGGER = logging.getLogger(__name__)
+from .const import LEASES_REGEX, UNKNOWN_DEVICE_NAME, Device
 
-_LEASES_REGEX = re.compile(
-    r"(?P<ip>([0-9]{1,3}[\.]){3}[0-9]{1,3})"
-    + r"\smac:\s(?P<mac>([0-9a-f]{2}[:-]){5}([0-9a-f]{2}))"
-    + r"\svalid\sfor:\s(?P<timevalid>(-?\d+))"
-    + r"\ssec"
-)
+_LOGGER = logging.getLogger(__name__)
```

Comment:
It also can be marked final to have it protected from overwriting.
```suggestion
_LOGGER: Final = logging.getLogger(__name__)
```